content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/README.md

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright 2019 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Modal (v1)
+====
+Modal component written in HTL that renders a configurable modal view for the content.
+
+## Features
+* Displays Content Fragment & Experience Fragment in Modal View.
+* Supports Modal View on page load & on click of any link.
+
+### Use Object
+The Modal component uses the `com.adobe.cq.wcm.core.components.models.Modal` Sling model as its Use-object.
+
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Modal component and are expected to be available as `Resource` properties:
+
+1. `./modalId` - represents the hash generated for the component path. Field is non-editable & shows up on page in author mode.
+2. `./description` - allows to provide description of the modal. This descirption value shows up on page in author mode.
+3. `./showModalByDefault` - allows you to enable/disable viewing of current modal on page load without providing #modalId in url.
+4. `./fragmentType` - this select field allows author to choose either content fragment or experience fragment options.

```

Comment:
As I mentioned in the interface, we should have just a `path`, no need to have three properties here.

Reply:
I'd suggest to call it `fragmentPath` as it is the property name for both experience and content fragment.